src/popup/components/Register/index.tsx

```diff
@@ -66,53 +69,34 @@ const ActionContainer = styled.div`
 type RegisterProps = {
   loading: boolean;
   onNext: (password: string) => void;
-  onClickTerms: () => void;
-  onClickPrivacyPolicy: () => void;
   error: string;
 };
 
-function Register(props: RegisterProps) {
-  const { loading, onNext, onClickTerms, onClickPrivacyPolicy } = props;
+const MIN_LENGTH = 8;
 
-  const minLength = 8;
-  const [newPassword, setNewPassword] = useState("");
-  const [confirmPassword, setConfirmPassword] = useState("");
-  const [hint, setHint] = useState(`Minimum length: ${minLength}`);
+function passwordError(password: string): string | undefined {
+  if (password.length < MIN_LENGTH)
+    return `Must be at least ${MIN_LENGTH} characters`;
+  return;
+}
 
-  const onClick = () => {
-    if (arePasswordsValidAndEquals && !loading) {
-      onNext(newPassword);
-    }
-  };
+function confirmError(password: string, confirm: string): string | undefined {
+  if (confirm !== password) return "Confirmation does not match";
```

Comment:
```suggestion
  if (confirm !== password) return "Password does not match";
```

[Looked around](https://www.google.com/search?q=password+doesn%27t+match+message&tbm=isch) a bit, also found [this](https://ux.stackexchange.com/a/35306), I think this phrasing is clearer and more common.

Reply:
Done.